packages/frontend/src/App.vue

```diff
@@ -664,6 +667,7 @@ export default {
   },
   mounted() {
     this.init();
+    utils.removeClass("workaround", "material-icons");
```

Comment:
please consider to add a 'TODO' comment to remove that stuff when root cause will solved